vendor/volk/volk.BUILD

```diff
@@ -9,5 +9,6 @@ cc_library(
         "@bazel_tools//src/conditions:windows": ["VK_USE_PLATFORM_WIN32_KHR"],
         "//conditions:default": [],
     }),
+    linkstatic = 1,
```

Comment:
Does this actually fix the downstream problems where we need to use `linkstatic=1`? In my experiments it doesn't have any effect.

Reply:
It does get downstream dependencies to pull in libvolk.a, which seemed to fix them just fine.
Not that it did much good, though, because the next thing that I ran into was that in order to use Vulkan, our Vulkan driver helpfully loaded LLVM from the system, which immediately collided with our built LLVM library's symbols and broke everything in subtle ways that were all sorts of fun to debug.  So we still need linkstatic=1 on our tests, and we should probably come up with a better story for managing LLVM DLL collisions at some point.